SPEC.md

```diff
@@ -491,6 +491,7 @@ JSON Schema for the Image Manifest
     * **eventHandlers** are optional, and should be a list of eventHandler objects. eventHandlers allow the app to have several hooks based on lifecycle events. For example, you may want to execute a script before the main process starts up to download a dataset or backup onto the filesystem. An eventHandler is a simple object with two fields - an **exec** (array of strings, ACE can append or override), and a **name** (there may be only one eventHandler of a given name), which must be one of:
         * **pre-start** - will be executed and must exit before the long running main **exec** binary is launched
         * **post-stop** - if the main **exec** process is killed then this is ran. This can be used to cleanup resources in the case of clean application shutdown, but cannot be relied upon in the face of machine failure.
+    * **workingDirectory** the working directory of the launched application, relative to the application image's root (optional string, must be absolute path, defaults to "/", ACE can override). If the directory does not exist in the application's assembled rootfs, it must be created by the ACE at runtime.
```

Comment:
I do not think the ACE should create the directory since then we have to set the permissions, etc. If the workingDirectory does not exist then it should fail.